src/object/srv_obj.c

```diff
@@ -1295,6 +1385,7 @@ ds_obj_enum_handler(crt_rpc_t *rpc)
 	/* for KEY2BIG case, just reuse the oeo_size to reply the key len */
 	if (rc == -DER_KEY2BIG)
 		oeo->oeo_size = enum_arg.kds[0].kd_key_len;
+	oeo->oeo_epoch = oeo->oeo_epoch;
```

Comment:
oeo_epoch set as itself?

Reply:
Thanks for catching that. It should be "= oei->oei_epoch;".